src/ResourceManager/IotCentral/Commands.IotCentral/Az.IotCentral.psd1

```diff
@@ -0,0 +1,131 @@
+#
+# Module manifest for module 'PSGet_AzureRM.IotCentral'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 5/5/2017
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+# RootModule = ''
+
+# Version number of this module.
+ModuleVersion = '0.1.0'
+
+# Supported PSEditions
+CompatiblePSEditions = 'Core', 'Desktop'
+
+# ID used to uniquely identify this module
+GUID = '042f1373-72e2-414e-9334-74d15ff4e6dc'
+
+# Author of this module
+Author = 'Microsoft Corporation'
+
+# Company or vendor of this module
+CompanyName = 'Microsoft Corporation'
+
+# Copyright statement for this module
+Copyright = 'Microsoft Corporation. All rights reserved.'
+
+# Description of the functionality provided by this module
+Description = 'Microsoft Azure PowerShell - Compute service cmdlets for Azure Resource Manager in PowerShell and PowerShell Core.  Manages virtual machines, hosted services, and related resources in Azure Resource Manager'
+
+# Minimum version of the Windows PowerShell engine required by this module
+PowerShellVersion = '5.1'
+
+# Name of the Windows PowerShell host required by this module
+# PowerShellHostName = ''
+
+# Minimum version of the Windows PowerShell host required by this module
+# PowerShellHostVersion = ''
+
+# Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+DotNetFrameworkVersion = '4.7.2'
+
+# Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+# CLRVersion = ''
+
+# Processor architecture (None, X86, Amd64) required by this module
+# ProcessorArchitecture = ''
+
+# Modules that must be imported into the global environment prior to importing this module
+RequiredModules = @(@{ModuleName = 'Az.Profile'; ModuleVersion = '0.3.0'; })
+
+# Assemblies that must be loaded prior to importing this module
+RequiredAssemblies = @('.\Microsoft.Azure.Management.IotCentral.dll')
+
+# Script files (.ps1) that are run in the caller's environment prior to importing this module.
+# ScriptsToProcess = @()
+
+# Type files (.ps1xml) to be loaded when importing this module
+
+# Format files (.ps1xml) to be loaded when importing this module
+
+# Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
+NestedModules = @('.\Microsoft.Azure.Commands.IotCentral.dll')
+
+# Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
+FunctionsToExport = @()
+
+# Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
+CmdletsToExport = @('New-AzureRmIotCentralApp', 'Get-AzureRmIotCentralApp', 'Set-AzureRmIotCentralApp', 'Remove-AzureRmIotCentralApp')
```

Comment:
These should be "Verb-AzNoun" rather than "Verb-AzureRmNoun"

Reply:
@maddieclayton I don't have to redefine my cmdlet file for that right? .NetCore will just use different constants (Az vs AzureRM) than .NetFramework right?

Reply:
Done